presto-base-jdbc/pom.xml

```diff
@@ -103,12 +103,6 @@
             <scope>test</scope>
         </dependency>
 
-        <dependency>
-            <groupId>org.jdbi</groupId>
-            <artifactId>jdbi</artifactId>
-            <scope>test</scope>
-        </dependency>
```

Comment:
How is this possible that there were unused dependency? I thought that such dependency causes build to fail.

Reply:
I’m not sure either, but I’ve noticed the dependency checker doesn’t always work for correctly tests dependencies.